falcon/media/json.py

```diff
@@ -8,7 +8,7 @@
 
 
 class JSONHandler(BaseHandler):
-    """Handler built using Python's :py:mod:`json` module."""
+    """Handler built using Python's :py:mod: `ujson` if available, :py:mod: `json` otherwise."""
```

Comment:
This would probably be better phrased 
```
Handler built using Python's :py:mod: `json` by default, but will use :py:mod: `ujson` if available.
```